core/src/test/java/apoc/export/csv/ExportCsvUseTypeTest.java

```diff
@@ -0,0 +1,80 @@
+package apoc.export.csv;
+
+import apoc.graph.Graphs;
+import apoc.util.TestUtil;
+import apoc.util.Util;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.neo4j.configuration.GraphDatabaseSettings;
+import org.neo4j.graphdb.Label;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.test.rule.DbmsRule;
+import org.neo4j.test.rule.ImpermanentDbmsRule;
+
+import static apoc.ApocConfig.APOC_EXPORT_FILE_ENABLED;
+import static apoc.ApocConfig.apocConfig;
+import static apoc.export.csv.ExportCsvTest.assertResults;
+import static apoc.export.csv.ExportCsvTest.readFile;
+import static apoc.util.MapUtil.map;
+import static apoc.util.TestUtil.testCall;
+import static org.junit.Assert.assertEquals;
+
+// Created to not affect ExportCsvTest results
+public class ExportCsvUseTypeTest {
+
+    @ClassRule
+    public static DbmsRule db = new ImpermanentDbmsRule()
+            .withSetting(GraphDatabaseSettings.load_csv_file_url_root, ExportCsvTest.directory.toPath().toAbsolutePath());
+
+
+    @BeforeClass
+    public static void setUp() {
+        TestUtil.registerProcedure(db, ExportCSV.class, Graphs.class);
+        apocConfig().setProperty(APOC_EXPORT_FILE_ENABLED, true);
+
+        db.executeTransactionally("CREATE (n:SuperNode { one: datetime('2018-05-10T10:30[Europe/Berlin]'), two: time('18:02:33'), three: localtime('17:58:30'), \n" +
+                "four: localdatetime('2021-06-08'), five: date('2020'), six: duration({months: 5, days: 1.5}), seven : '2020'}) \n" +
+                "WITH n CREATE (n)-[:REL_TYPE {rel: point({x: 56.7, y: 12.78, crs: 'cartesian'})}]->(m:AnotherNode)");
```

Comment:
I think it's better not to include arrays at the moment 
since there's another specific card for arrays (the one with id `F9oIci9k`), 
so I could put them there in case